src/components/Home/Search/Search.jsx

```diff
@@ -1,5 +1,17 @@
-const Search = () => {
-  return <div>Search</div>;
+const Search = ( { setSearchValue } ) => {
+
+
+  return (
+    <div>
+      <form>
+        <input className="search"
+                type="text" 
+                placeholder="Search Card..." 
+                onChange={(event) => {setSearchValue(event.target.value)}} 
+        />
```

Comment:
You should add onSubmit={(event) => {event.preventDefault()}, to prevent the page from refreshing when you press enter.